src/networking/ClientConnection.java

```diff
@@ -29,50 +31,47 @@ public void startConnection(PeerInfoConfig peerInfoConfig) throws Exception {
         out.flush();
         in = new DataInputStream(socket.getInputStream());
 
-        //TODO - create handshake message and send with outputMessage(Message m)
+        //FIXME: Is this the *other* peerID? Aka the peerID that we started a connection with? Or is that serverPeerID?
+        HandshakeMessage handshakeMessage = new HandshakeMessage(peerID);
```

Comment:
@nt92 Help fam

Reply:
No I think this would be this peerID since the message we are sending is the handshake from the current client, aka it should have that peer's id in the message be sent

Reply:
Ok cool that's what I thought. Either way we'll be able to find out if this is a bug pretty easily.